Custom Pages for `<UserProfile />` and `<OrganizationProfile />` (#404)
clerkinc/clerk-docs: docs/references/javascript/clerk/organization-profile.mdx

function closeOrganizationProfile(): void;
```

## Custom Pages

> Organization profile can be customized by adding custom pages to it. Custom pages are added to the organization profile using the `customPages` prop.

Reviewer: ```suggestion
You can add customized pages for the `<OrganizationProfile />` component by using the `customPages` prop.

```
